src/main/java/redis/clients/jedis/commands/AdvancedBinaryJedisCommands.java

```diff
@@ -64,5 +65,7 @@
 
   List<byte[]> aclCat(byte[] category);
 
+  String aclLog(byte[] options);
```

Comment:
Why not `byte[] aclLog(byte[] options)`?

Reply:
I looked at various methods in the BinaryJedisCommand, and still not sure what is the *rule*

Many of the methods are return the status as a string for example `set`,`type` `setex`, `psetex`...

*Since I do not know how the binary methods are used I chose the simplity*

Reply:
In BinaryJedis, we should be able to do most if not all operations using `byte[]` instead of String. The common exceptions, in case of parameters, are IP, password, etc. And common exceptions, in case of return types, are definitive Strings i.e. `"OK"` in set, setex, psetex, etc. A case can be made for `type`. But that's historical implementation and the users don't seem to bothered about it because isn't any issue submitted about it.

Reply:
Done.